lms/templates/instructor/instructor_dashboard_2/course_info.html

```diff
@@ -5,7 +5,7 @@
 %>
 
 %if settings.FEATURES.get('DISPLAY_ANALYTICS_ENROLLMENTS') or section_data.get('enrollment_message'):
-  <h3 class="hd hd-3">${_("Enrollment Information")}</h3>
+  <h4 class="hd hd-4">${_("Enrollment Information")}</h4>
```

Comment:
I found it on the sandbox, and the styling seems OK. I have a general question though about why we use header classes when the styling matches the actual header tag level.